diff --git a/LS_COLORS b/LS_COLORS
index 6a747d8..baa6752 100644
--- a/LS_COLORS
+++ b/LS_COLORS
@@ -31,10 +31,10 @@
 # with this program.  If not, see <http://www.perlfoundation.org/artistic_license_1_0>.
 
 # core {{{1
-BLK                   38;5;68
+BLK                   38;5;30
 CAPABILITY            38;5;17
 CHR                   38;5;113;1
-DIR                   38;5;30
+DIR                   38;5;75;1
 DOOR                  38;5;127
 EXEC                  38;5;208;1
 FIFO                  38;5;126
